drivers/ovsd/ovsSwitch.go

```diff
@@ -100,6 +101,10 @@ func NewOvsSwitch(bridgeName, netType, localIP, fwdMode string,
 	sw.netType = netType
 	sw.uplinkDb = cmap.New()
 	sw.hostPvtNW = hostPvtNW
+	sw.vxlanEncapMtu, err = netutils.GetHostLowestLinkMtu()
+	if err != nil {
+		log.Fatalf("Failed to get Host Node MTU. Err:%v", err)
```

Comment:
nitpick: add a space between `:` and `%v`

Reply:
ok;